From: SeongJae Park <sj@kernel.org>
Date: Sun, 22 Sep 2024 09:26:13 -0700
Subject: [PATCH] Docs/mm/damon: add links to DAMON academic papers

There are two academic papers that presenting DAMON.  Specifically, the
paper for Middleware'19 Industry [1] describes the monitoring mechanism
of DAMON and access pattern profiling-guided optimizations.  The paper
for HPDC'22 extends the coverage to DAMOS and user-space driven
auto-tuning approach, namely DAMOOS.

Some people like researchers would prefer reading such academic papers
over kernel documentation.  It is written in the academic papers' format
and reviewed-by the academic community.  Also, those are better to be
cited on other academic papers.  Such citation helps tracking of related
works.

Add a paragraph introducing those to DAMON kernel doc.

[1] https://dl.acm.org/doi/abs/10.1145/3366626.3368125
[2] https://dl.acm.org/doi/abs/10.1145/3502181.353146

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 Documentation/mm/damon/index.rst | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/Documentation/mm/damon/index.rst b/Documentation/mm/damon/index.rst
index dafd6d028924..f109d1de8ede 100644
--- a/Documentation/mm/damon/index.rst
+++ b/Documentation/mm/damon/index.rst
@@ -37,3 +37,8 @@ with no code but simple configurations.
 
 To utilize and control DAMON from the user-space, please refer to the
 administration :doc:`guide </admin-guide/mm/damon/index>`.
+
+If you are more familiar with academic papers, please refer to and/or cite
+DAMON papers for `Middleware19 Industry
+<https://dl.acm.org/doi/abs/10.1145/3366626.3368125>`_ and `HPDC'19
+<https://dl.acm.org/doi/abs/10.1145/3502181.3531466>`.
-- 
2.39.5

